treestats_paper.tex

```diff
@@ -931,6 +955,12 @@ \section*{Application to 1000 Genomes tree sequences}
     }
 \end{figure}
 
+\krt{Hmmm...not sure about this paragraph.  Under strict BGS, some of what is stated here doesn't hold.  The tree will
+    still be neutral in shape, just shorter overall.  For the case of many weakly-deleterious segregating mutations, the
+    overall shape of the tree may not be that different from neutral expectations (Neuhaueser/Krone and the later
+Przeworksi paper, all in Genetics IIRC?) It also seems to me that the idea of what branch-vs-site should look like under
+these processes vs what they should look like in a ts inferred from data where the true model is these processes are
+getting conflated.}
```

Comment:
perhaps we need to add the caveats that (a) BGS may not distort tree shape very much (although other selection will do more), and (b) there's also inference to worry about?

Reply:
yes, will try to make some edits along these lines.